djcelery/management/commands/celery.py

```diff
@@ -17,7 +17,6 @@ class Command(CeleryCommand):
         preload_options = base.preload_options if base.preload_options else []
     else:
         preload_options = []
-    preload_options = base.preload_options if base.preload_options else []
```

Comment:
It looks like this and the if/else block could be replace with just:

    preload_options = getattr(base, 'preload_options', []) or []

Reply:
I fixed it. Thanks, @liquidpele :).